include/abb_librws/rws_client.h

```diff
@@ -278,6 +278,18 @@ class RWSClient : public POCOClient
     std::vector<SubscriptionResource> resources_;
   };
 
+  /**
+   * \brief An enumeration of controller coordinate frames.
+   */
+  enum class Coordinate
+  {
+    BASE,
+    WORLD,
+    TOOL,
+    WOBJ,
+    ACTIVE
```

Comment:
Each item in the enum should have a short comment ([example](https://github.com/ros-industrial/abb_librws/blob/4741d83b865a1a495d9973294607d65c660d2f21/include/abb_librws/rws_rapid.h#L50-L59)).